install_config/persistent_storage/pod_security_context.adoc

```diff
@@ -91,39 +87,36 @@ spec:
         server: example.com
         path: "/nfsexport"
 ----
-
 ====
 
-All containers in the above pod will be members of the group 1234 and
-will have access to the volume regardless of what the container
-runs as.
-
-This is useful with volumes which do not allow you to change the
-export's ownership on the client such as NFS and GlusterFS.
+All containers in the above pod will be members of the group *1234* and will
+have access to the volume regardless of what the container runs as. This is
+useful for volumes which do not allow you to change the export's ownership on
+the client, such as NFS and GlusterFS.
 
 == FSGroup
 
-`FSGroup` stands for file system group. It is a supplemental group as
-defined above but with some extra functionality associated with it. If
-`FSGroup` is specified it will have the following effects on volumes
-which support ownership management:
+`*FSGroup*` stands for file system group. It is a supplemental group as defined
+above, but with some extra functionality. If `*FSGroup*` is specified, it will
+have the following effects on volumes which support ownership management:
 
-* The owning group of the volume will be set to the specified `FSGroup`.
-* Newly created files in the volume will also be owned by `FSGroup`.
-* Read and write permissions will be given to `FSGroup`
-* The specified `FSGroup` will be added to the pods list of supplemental groups.
+* The owning group of the volume will be set to the specified `*FSGroup*`.
+* Newly created files in the volume will also be owned by `*FSGroup*`.
+* Read and write permissions will be given to `*FSGroup*`.
+* The specified `*FSGroup*` will be added to the pods list of supplemental
+groups.
 
-In effect this provides access to your pod, and any other pods with the
-same `FSGroup` access to the volume. 
+This provides access to your pod and any other pods with the same `*FSGroup*`
+access to the volume.
 
```

Comment:
how about: "This gives your pod, and any other pods with the same `*FSGroup*`, access to the volume" ?